.github/actions/setup/action.yml

```diff
@@ -2,16 +2,16 @@ name: Setup
 runs:
   using: composite
   steps:
-    - uses: pnpm/action-setup@v2
-      with:
-        version: 8.6.0
-
```

Comment:
@Zizzamia It doesn't seem like this is needed?

Reply:
Yeah your right, great catch!